lte/protos/BUILD.bazel

```diff
@@ -354,6 +354,16 @@ cpp_grpc_library(
     deps = [":sms_orc8r_cpp_proto"],
 )
 
+proto_library(
```

Comment:
Tested locally that
`bazel build //lte/protos:diam_errors_proto` and
`bazel build //lte/protos:diam_errors_cpp_proto`
are building. This is the only bazel change here though. These targets are currently not used in any bazel binary or test target. I recommend to remove these changes for now. The bazel proto libraries should be added when `diam_errors.proto` is used in other bazel targets - fyi using bazel is an ongoing effort.